Resources/doc/reference/installation.rst

```diff
@@ -151,7 +151,7 @@ if you use the API.
                     - sonata.formatter.twig.gist
                     - sonata.media.formatter.twig
 
-* Create configuration file ``sonata_block.yml`` for block rendering:
+* Create configuration file ``sonata_block.yml`` for block rendering as per [Sonata Block Configuration]:
```

Comment:
That obvious step is also skipped in the documentation after the first example. ie, 

The import example is given for the first config/yml only at https://github.com/bizmate/SonataNewsBundle/blame/d5856b005690283ac152492f777a20ce27492f35/Resources/doc/reference/installation.rst#L92

In the other configs is not included and it becomes repetitive. If added should be either left only once at the top or put everywhere a config is described. For instance there is not need to repeat that the import is required here
https://github.com/bizmate/SonataNewsBundle/blame/d5856b005690283ac152492f777a20ce27492f35/Resources/doc/reference/installation.rst#L105

or 

https://github.com/bizmate/SonataNewsBundle/blame/d5856b005690283ac152492f777a20ce27492f35/Resources/doc/reference/installation.rst#L122